src/api/test/test_helper.rb

```diff
@@ -294,7 +259,7 @@ def current_user
       Capybara.current_driver = :rack_test
 # crude work around - one day I will dig into why this is necessary
       Minitest::Spec.new('MINE') unless Minitest::Spec.current
-      self.class.start_test_api
+      Suse::Backend.start_test_backend
       #Capybara.current_driver = Capybara.javascript_driver
```

Comment:
I've tried it without it, it didn't work, tests where failing all over the place.